### PR TITLE
Visual alert for IV drip

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Adds a visual alert for your mob having an IV connected.

image

## Why It's Good For The Game

It's easy to forget you have one of these in your arm until it's ripped and you're bleeding out.

## Changelog

:cl: LT3
qol: Being connected to an IV now has a visual indicator
/:cl:

### CODE REVIEW

File: code/game/machinery/iv_drip.dm [hunk 1]
```diff
@@ -314,6 +314,8 @@
 	var/datum/reagents/container = get_reagents()
 	log_combat(usr, target, "attached", src, "containing: ([container.get_reagent_log_string()])")
 	add_fingerprint(usr)
+	ADD_TRAIT(target, IV_CONNECTED_TRAIT, REF(src))
```
Comment (Mothblocks) [inline code/game/machinery/iv_drip.dm:317]: What's the point of this trait if nothing is reading from it?

File: code/_onclick/hud/alert.dm [hunk 2]
```diff
@@ -877,6 +877,11 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 	carbon_owner.changeNext_move(CLICK_CD_RESIST)
 	carbon_owner.shoes.handle_tying(carbon_owner)
 
+/atom/movable/screen/alert/iv_connected
+	name = "IV Connected"
+	desc = "You have an IV connected to your arm. Remember to remove it or drag the IV stand with you before moving, or else it will rip out!"
+	icon_state = ALERT_IV_CONNECTED
```
Comment (Mothblocks) [inline code/_onclick/hud/alert.dm:883]: I think this should exist just in the `iv_drip` file, then you can have `ALERT_IV_CONNECTED` only exist in that file

File: code/game/machinery/iv_drip.dm [hunk 3]
```diff
@@ -314,6 +314,8 @@
...
+	target.throw_alert("iv_connected", /atom/movable/screen/alert/iv_connected)
```
Comment (Mothblocks) [inline code/game/machinery/iv_drip.dm:318]: Use `ALERT_IV_CONNECTED` instead of "iv_connected" in both places

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/_onclick/hud/alert.dm (hunk 2 above)
Comment (lessthnthree) [reply code/_onclick/hud/alert.dm:883]: Sure

**Pull request merged**